torchci/lib/drciUtils.ts

```diff
@@ -107,3 +108,58 @@ ${sev_list}\n
 `;
   }
 }
+
+// The context here is the context from probot.
+// Today we only use probot for upserts, but this could later be split into logger
+export async function upsertDrCiComment(owner: string, repo: string, prNum: number, context: any, prUrl: string) {
+  // Dr.CI only supports pytorch/pytorch at the moment
+  context.log("It's me!")
+  if (owner != OWNER || repo != REPO) {
```

Comment:
We have `isPyTorchPyTorch(owner, repo)` function in `utils.ts`

Reply:
Nice, I'll switch to that